analysis/webservice/algorithms/doms/BaseDomsHandler.py

```diff
@@ -196,14 +221,24 @@ def __addDynamicAttrs(csvfile, executionId, results, params, details):
             for match in primaryValue['matches']:
                 platforms.add(match['platform'])
 
+        # insituDatasets = params["matchup"].split(",")
+        insituDatasets = params["matchup"]
+        insituLinks = set()
+        for insitu in insituDatasets:
+            if insitu == "samos":
+                insituLinks.add("http://samos.coaps.fsu.edu/html/nav.php?s=2")
+            if insitu == "icoads":
+                insituLinks.add("https://rda.ucar.edu/datasets/ds548.1/")
+            if insitu == "spurs":
+                insituLinks.add("https://podaac.jpl.nasa.gov/spurs")
+
         global_attrs = [
             {"Global Attribute": "Platform", "Value": ', '.join(platforms)},
             {"Global Attribute": "time_coverage_start",
              "Value": params["startTime"].strftime(ISO_8601)},
             {"Global Attribute": "time_coverage_end",
              "Value": params["endTime"].strftime(ISO_8601)},
-            # TODO I don't think this applies
-            # {"Global Attribute": "time_coverage_resolution", "Value": "point"},
+            {"Global Attribute": "time_coverage_resolution", "Value": "point"},
```

Comment:
Did you mean to enable this? "point" doesn't make sense as a value for this attribute. The definition is:

> time_coverage_resolution | Describes the targeted time period between each value in the data set. Use ISO 8601:2004 duration format, preferably the extended format as recommended in the Attribute Content Guidance section.

Reply:
Removing all incorrect "point" references